packages/scripts/webapp/scripts/start.js

```diff
@@ -10,7 +10,6 @@ module.exports = function start(env, _, options) {
 			'--inline',
 			'--config',
 			hereRelative(__dirname, '../config/webpack.config.js'),
-			'--open',
 			'--progress',
 			'--display-error-details',
```

Comment:
we should avoid mix CLI args and config file, it's a mess.

Reply:
I remove `inline` (it's the default)
`progress` is cli only
`display-error-details` ... I don't find any documentation on that :/